### PR TITLE
Map settings

### PR DESCRIPTION
**Pull request by dhakelila**
This PR implements map first settings. 

You can now see targets and indicators into dashboard, and a "fake layer" appearing when clicking over a target or an indicator. 

These are only the first steps. I keep going.

### CODE REVIEW

File: js/src/templates/map/targets.hbs [hunk 1]
```diff
@@ -1,24 +1,10 @@
...
-    <div class="indicator-wrapper">
-      <input type="radio" name="indicator" value="percentage_of_people_violence">
-      <label for="percentage_of_people_violence"><h1 class="indicator-title">indicator name</h1></label>
-      <button class="js--indicator-info" data-slug="percentage_of_people_violence">i</button>
+{{#each targets}}
+  <div class="m-dashboard-target" id="{{this.slug}}">
+    <div class="header">
+      <button class="js--toggle-target btn-toggle-target" data-slug={{this.slug}}>+</button>
+      <input type="radio" name="layer-selector" value={{this.slug}} id={{this.slug}} class="js--layer-selector -target">
+      <label for={{this.slug}}><h2 class="target-title js--target" data-slug={{this.slug}}>{{this.title}}</h2></label>
```
Comment (agnlez) [inline js/src/templates/map/targets.hbs]: When you are in a Handlebars loop you don't need to use "this" to access to the property, you can access directly. Replace  {{ this.title }}  to {{ title }} and so on.

File: js/src/views/map/dashboard.js [hunk 2]
```diff
@@ -4,46 +4,98 @@ var $ = require('jquery'),
...
+    var indicatorsGrupedByType = this.indicatorsCollection.groupByType();
+
+    this.$('#' + targetSlug).find('.js--indicators').html(indicatorsTemplate(indicatorsGrupedByType))
+  },
+
+  _selectLayer: function(e) {
+    var $currentTarget = $(e.currentTarget);
+    var currentTargetSlug = $currentTarget.val();
+
+    if ($currentTarget.hasClass('-target')) {
```
Comment (andresgnlez) [inline js/src/views/map/dashboard.js]: What, what..?

File: js/src/views/map/dashboard.js [hunk 3]
```diff
@@ -4,46 +4,98 @@ var $ = require('jquery'),
...
+      this._setActiveLayer('target', currentTargetSlug);
+    } else {
+      this._setActiveLayer('target', currentTargetSlug);
+    }
   },
```
Comment (agnlez) [inline js/src/views/map/dashboard.js:94]: What, what..?

File: js/src/views/map/map.js [hunk 4]
```diff
@@ -18,48 +21,120 @@ var MapView = Backbone.View.extend({
       worldCopyJump: true
     },
     cartodb: {
-      user_name: 'dhakelila',
-      type: 'cartodb',
+      user_name: 'goal16',
```
Comment (agnlez) [inline js/src/views/map/map.js]: Wouldn't be better use the CONFIG json file for these cases? I think that's the purpose of this.

File: js/src/views/map/map.js [hunk 5]
```diff
@@ -18,48 +21,120 @@ var MapView = Backbone.View.extend({
...
+      //This way, we are sure we only have one layer at a time.
+
+      this._addLayer();
+    }, this));
+  },
+
+  _createLayer: function() {
+    var sql = this._getLayerQuery();
+    var cartoAccount = this.options.cartodb.user_name;
+    var cartoCss = this.options.cartodb.css || '#table_score_test{ polygon-fill: #FF6600; polygon-opacity: 0.7; line-color: #FFF; line-width: 0.5; line-opacity: 1; }';
```
Comment (agnlez) [inline js/src/views/map/map.js:85]: I would move to a options object o CONFIG JSON, if you start to change map params, you have to check the whole view and that can be a little annoying.

File: js/src/views/map/map.js [hunk 6]
```diff
@@ -18,48 +21,120 @@ var MapView = Backbone.View.extend({
...
+  _getLayerQuery: function() {
+    var query;
+    var layer = this.status.get('layer');
+    var type = this.status.get('layerType');
+
+    if (type === 'target') {
+      query = 'SELECT * FROM score_test WHERE indicator_slug=' + layer;
+    } else {
+      query = 'SELECT * FROM score_test WHERE indicator_slug=' + layer;
+    }
```
Comment (agnlez) [inline js/src/views/map/map.js:137]: What, what..? x2

File: js/src/views/map/map.js [hunk 7]
```diff
@@ -18,48 +21,120 @@ var MapView = Backbone.View.extend({
...
+
+    return 'SELECT * FROM score_test';
```
Comment (agnlez) [inline js/src/views/map/map.js:139]: O.o Does it make any sense this?

File: _sass/modules/_m-dashboard-indicators.scss [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+      border-radius: 50%;
+      background-color: $bg-color-1;
+      border: 1px solid $color-4;
+    }
+
+    .indicator-title {
+      font-size: rem(12px);
+      line-height: rem(20px);
+      color: $font-color-3;
+      font-size: rem(14px);
```
Comment (agnlez) [inline _sass/modules/_m-dashboard-indicators.scss:33]: Now we have many font-size-related variables, use them, please.

File: _sass/modules/_m-dashboard-indicators.scss [hunk 9]
```diff
@@ -0,0 +1,36 @@
+.m-dashboard-indicators {
+
+  .indicators-wrapper {
+    margin-left: rem(60px);
+  }
+
+  .indicator-type {
+    display: block;
+    padding-bottom: rem(7px);
+    border-bottom: 2px solid $color-3;
```
Comment (agnlez) [inline _sass/modules/_m-dashboard-indicators.scss:10]: rem, rem!

File: _sass/modules/_m-dashboard-indicators.scss [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+
+    input {
+      width: rem(16px);
+      height: rem(16px);
+      border-radius: 50%;
+      background-color: $bg-color-1;
+      border: 1px solid $color-4;
```
Comment (agnlez) [inline _sass/modules/_m-dashboard-indicators.scss:26]: Don't forget about transform border width to rem!

**Pull request merged**